openshift/templates/visual-verifier/visual-verifier-deploy.prime.test.param

```diff
@@ -3,13 +3,13 @@
 # Component: .
 # Template File: ../openshift/templates/visual-verifier/visual-verifier-deploy.json
 #=========================================================
-# NAME=visual-verifier
-# APP_NAME=issuer
+# NAME=visual-verifier-prime
+# APP_NAME=issuer-prime
 # APP_GROUP=prime
-# SUFFIX=-prime
+# SUFFIX=
```

Comment:
Why did we switch from suffixed names here?

Reply:
must have accidentally changed it while doing sandbox. Change reverted